GeoTIFF_Standard/standard/annex-b.adoc

```diff
@@ -355,9 +355,14 @@ An exact geodetic definition of geodetic datums and reference frames is beyond t
 A geodetic coordinate reference system is created by associating a coordinate system - a set of axes - with a geodetic datum.
 Subtypes of geodetic CRS supported by GeoTIFF are as follows.
 
-* Feocentric, when the coordinate system is a 3-dimensional Cartesian coordinate system with its origin at or near the centre of the earth. The Z-axis is in or parallel to the earth's axis of rotation. The X-axis is in the plane of the equator and passes through its intersection with the prime meridian, and the Y-axis is in the plane of the equator forming a right-handed coordinate system with the X and Z axes.
+* Geocentric, when the coordinate system is a 3-dimensional Cartesian coordinate system with its origin at or near the centre of the earth.
+  The Z-axis is in or parallel to the earth's axis of rotation.
```

Comment:
Note: the typo fixed by the PR is the first character. The rest is the long line splitted in shorter lines, with no  change in the text.